lighthouse-core/test/lib/dependency-graph/simulator/simulator-test.js

```diff
@@ -260,20 +278,20 @@ describe('DependencyGraph/Simulator', () => {
     describe('on a real trace', () => {
       const trace = require('../../../fixtures/traces/progressive-app-m60.json');
       const devtoolsLog = require('../../../fixtures/traces/progressive-app-m60.devtools.log.json');
-      let result;
 
-      beforeAll(async () => {
+      it('should compute a timeInMs', async () => {
```

Comment:
cleaned this up because it made looking at just one test with `.only` impossible during development